Cloud reference overhaul (#64)
crate/cloud-reference: docs/overview.rst

Organization
============

> The organization is the highest structure in yout CrateDB Cloud Console.

Reviewer: ```suggestion
The organization is the highest structure in your CrateDB Cloud Console.
```

> Multiple clusters and users can exist in a organization at any moment. For the

Reviewer: ```suggestion
Multiple clusters and users can exist in a organization at any moment. For 
```

first-time users, an organization called "My organization" is automatically 

> crated upon first login.

Reviewer: ```suggestion
created upon first login.
```

To see list of all the clusters in your organization, go to the Account
tab in your left-hand menu:

.. image:: _assets/img/organization-dashboard.png
   :alt: Cloud Console organization overview

The Organization overview consists of seven tabs: *Clusters*, *Settings*,
*Users*, *Billing*, *Payment Methods*, *Audit Logs*, and *Regions*. By default

> you are brought to the Clusters tab, which provides quick overview of all

Reviewer: ```suggestion
you are brought to the Clusters tab, which provides a quick overview of all
```

To learn more about user roles and their meaning, see our documentation on
`user roles`_.

.. _overview-org-billing:

Organization Billing
--------------------

The Billing tab shows all your existing subscriptions, along with which

> cluster is currently using the subscription. Current accumulated billing

Reviewer: ```suggestion
cluster is currently using the subscription. The current accumulated billing
```

.. image:: _assets/img/payment-methods2.png
   :alt: Cloud Console payment methods

.. _overview-org-audit:

Organization Audit Logs
-----------------------

> This tab shows the Audit Logs of current organization.

Reviewer: ```suggestion
This tab shows the Audit Logs of the current organization.
```

    stateful connections (e.g., JDBC) must be configured to send keep-alive
    heartbeat queries.

.. _overview-cluster-overview-next-steps:

Next Steps
~~~~~~~~~~

* **Connect to your cluster**: Click the "view code snippets" link to
  see a list of snippets for the available clients and libraries. These
  include: CLI clients, Java, JavaScript, PHP, Python, and Ruby.

* **Demo data**: Clicking the "import the demo data" will show you two
  snippets that can be used in the SQL Console of your Admin UI. The first

>   snippet a "nyc_taxi" table, and the second one will populate it with our 

Reviewer: ```suggestion
  snippet will create a "nyc_taxi" table, and the second one will populate it with our 
```

  demo dataset.

* **Backups**: The "see my backups" will take you to the Backups tab, where 
  you can see all your backups.

For more information on the CrateDB concepts used here, refer to the `CrateDB
architecture documentation`_ or the :ref:`glossary<glossary>`.

.. _overview-cluster-overview-access:

Access
------

The access tab contains your credentials, security settings, allowlist, and
information about private endpoints.

.. image:: _assets/img/cluster-access.png
   :alt: Cloud Console cluster access tab

.. _overview-cluster-overview-credentials:

Credentials
~~~~~~~~~~~

> Here you can change your CrateDB Console password.

Reviewer: ```suggestion
Here you can change your CrateDB database password.
```

.. _overview-cluster-overview-security:

Security
~~~~~~~~

Click the box labelled *Deletion Protected* to add deletion protection to your
cluster. This will ensure that it can only be deleted in either the CrateDB
Console or croud by a user with :ref:`org admin rights <org-roles>` (or by a

> Crate.io employee superuser).

Reviewer: This is not the case - deletion protection ensures that it cannot be deleted by anyone, unless deletion protection is switched off. The intent here is to make deletion a doubly-conscious choice and not accidentally delete your production cluster.

.. _overview-cluster-overview-allowlist:

Allowlist
~~~~~~~~~

By using the IP allowlisting feature, you can restrict access to your cluster
to an indicated IP address or `CIDR block`_. Click the blue *Add
Address* button and you can fill out an IP address or range and give it a
meaningful description. Click *Save* to store it or the bin icon to delete a
range. Keep in mind that once IP allowlisting has been set, you cannot access
the Admin UI for that cluster from any other address.

If no allowlist address or address range is set, the cluster is publicly
accessible by default. (Of course, the normal authentication procedures are
always required.) Only an :ref:`org admin <org-roles>` can change the
allowlist.

.. _overview-cluster-overview-private-endpoints:

Private endpoints
~~~~~~~~~~~~~~~~~

A private endpoint, or private link, is a mechanism that allows a secure, 
private connection to your cluster. Effectively, it allows you to bypass the

> public internet when accessing the environment where your cluster is deployed.

Reviewer: Maybe worth mentioning that it cannot be used across providers - i.e. from your Azure account to your AWS cluster.

graph panel are displayed in different colors. These values can also be read
directly by hovering over the relevant point on the time axis (the X axis).

.. _overview-cluster-settings-scale:

Scale
-----

On the Scale tab, current configuration of your cluster is shown. You can see
your current plan, resources of a single node, and overall resources of the
cluster.

.. image:: _assets/img/cluster-scale.png
   :alt: Cloud Console cluster scaling tab

You can scale your cluster by clicking the *Edit cluster configuration* button
in the top-right:

.. image:: _assets/img/cluster-scale-edit.png
   :alt: Cloud Console cluster scaling edit

Now you can do three different things:

- Change the plan of your cluster

> - Increase node's storage

Reviewer: ```suggestion
- Increase storage on each node
```

- Icrease/decrease the number of nodes

> You can do only one of those operations each time, i.e. you can't change plane

Reviewer: ```suggestion
You can do only one of those operations at a time, i.e. you can't change plans
```

    scale pricing shown here, but do not worry! This does not mean that your
    promotion or discount is not functioning.

.. WARNING::

    Storage capacity increases for a given cluster are irreversible. To reduce
    cluster storage capacity, reduce the cluster nodes instead (up to a
    minimum of 2, although we recommend maintaining a minimum of 3 for
    production use).

.. _overview-cluster-upgrade:

Upgrades
--------

The Upgrade cluster tab shows two things: the current version of the cluster
and, if a :ref:`minor upgrade <gloss-version>` or
:ref:`patch upgrade <gloss-version>` is possible, a list of available
upgrades. If no upgrade is possible, because the cluster is up to date with 
the latest version of CrateDB, nothing will be displayed in the Available
Upgrades panel. You can upgrade a cluster from the Available upgrades panel by

> clicking the *Upgrade* button next to a specific shown version. Upgrading may
> take a brief moment, during which other cluster operations will be disabled.

Reviewer: ```suggestion
clicking the *Upgrade* button next to a specific shown version. Depending on how much data you have in your cluster, upgrading might take a while, during which other cluster operations will be disabled. Your cluster's nodes
will be upgraded one at a time, so your cluster will remain available through the process.
```
